packages/components/icons/src/generated/AngleDownReact.tsx

```diff
@@ -139,11 +139,15 @@ const SvgAngleDown = (props: SVGProps) => (
     width={24}
     height={24}
     viewBox="0 0 24 24"
+    fill="none"
```

Comment:
This `fill="none"` property is a problem and I already had to remove it from the previous new icons.
I think we need to remove it again from all of them and let UX know so they can also remove it from the original icons we're importing.

Reply:
Ah okay thanks for the heads-up, will remove it and let Natalia know also.

Reply:
Here's the fix - [d557b34](https://github.com/commercetools/ui-kit/pull/2350/commits/d557b34039e25c05d65d2485911aa2506a80ec32).